silence/silence.go

```diff
@@ -571,36 +583,15 @@ func (s *Silences) toMeshSilence(sil *pb.Silence) *pb.MeshSilence {
 	}
 }
 
-func (s *Silences) setSilence(sil *pb.Silence, now time.Time, skipValidate bool) error {
-	sil.UpdatedAt = now
```

Comment:
Moved `sil.UpdatedAt` out of `setSilence` so when we check the limits in `CheckLimits` the full size of the `*pb.MeshSilence` is taken into account, not just the size of `*pb.Silence`.